core/snmp/profile-mapper/src/test/java/org/opennms/core/snmp/profile/mapper/SnmpProfileMapperIT.java

```diff
@@ -103,7 +113,7 @@ public void testSnmpProfileMapper() throws UnknownHostException {
                     null);
             Optional<SnmpAgentConfig> agentConfig = future.get();
             assertTrue(agentConfig.isPresent());
-            assertEquals(ttl, agentConfig.get().getTTL().longValue());
+           // assertEquals(ttl, agentConfig.get().getTTL().longValue());
```

Comment:
do we need the commented check?

Reply:
thanks didn't aware someone comment it out.